cmd/akamai-purger/main_test.go

```diff
@@ -0,0 +1,60 @@
+package notmain
+
+import (
+	"testing"
+	"time"
+)
+
+func TestThroughput_validate(t *testing.T) {
+	type fields struct {
+		ResponsesPerBatch int
+		PurgeBatchEvery   time.Duration
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		wantErr bool
+	}{
+		{"deployability defaults",
+			fields{
+				ResponsesPerBatch: 33,
+				PurgeBatchEvery:   10 * time.Second},
```

Comment:
Worth documenting where this value comes from, especially since the current `config/akamai-purger.json` has 350ms configured, not 10s.